docs/source/installation/plugins.rst

```diff
@@ -0,0 +1,72 @@
+.. _plugins:
+
+=======
+Plugins
+=======
+
+Plugins are features that extend Manim's core functionality. Since Manim is
+extensible and not everything belongs in its core, we'll go over how to
+install, use, and create your own plugins.
+
+*Note: The standard naming convention for plugins is to prefix the plugin*
+*with* ``manim-``
+
+Installing Plugins
+******************
+Plugins can be easily installed to your python site-packages via the ``pip``
+command:
+
+.. code-block:: bash
+
+    pip install manim-*
+
+The standard naming conventions makes plugins easy to find on organizations
+such as `PyPi.org <https://pypi.org/>`_. After installing plugins, you may
+use the ``manim plugins`` command to list and update Manim to import your
+plugins, see the following help output:
+
+.. code-block:: bash
+
+    manim plugins -h
```

Comment:
Typo: Inconsistent with the [plugin creation doc](https://pypi.org/project/manim-plugintemplate/).
![image](https://user-images.githubusercontent.com/65204531/100537056-3058e400-324b-11eb-9182-92df20a40b73.png)

Reply:
Thanks! Let me know if you find any others discrepancies. I'll hold off on any commits/publishing to PyPI until this gets more eyes.